Source/WebCore/bindings/scripts/test/JS/JSDOMWindowConstructors.cpp

```diff
@@ -0,0 +1,5 @@
+/*
+    This file is generated to inform build scripts that JSDOMWindowConstructors.h and
+    JSDOMWindowConstructors.cpp were created for DOMWindowConstructors.idl, and prevent the build
```

Comment:
`DOMWindowConstructors.idl` doesn't exist (since the file name wasn't renamed).

Reply:
ditto